src/main/resources/messages_it_IT.properties

```diff
@@ -551,10 +551,10 @@ login.oauth2AccessDenied=Accesso negato
 login.oauth2InvalidTokenResponse=Risposta token non valida
 login.oauth2InvalidIdToken=Id Token non valido
 login.userIsDisabled=L'utente è disattivato, l'accesso è attualmente bloccato con questo nome utente. Si prega di contattare l'amministratore.
-login.alreadyLoggedIn=You are already logged in to
-login.alreadyLoggedIn2=devices. Please log out of the devices and try again.
-login.toManySessions=You have too many active sessions
-login.toManySessions2=Please log out of the devices and try again. Alternatively, you can upgrade to Stirling PDF Pro.
+login.alreadyLoggedIn=Hai già effettuato l'accesso a
+login.alreadyLoggedIn2=dispositivi. Esci dai dispositivi e riprova.
+login.toManySessions=Hai troppe sessioni attive
```

Comment:
Actually let me remove these messages, decided not to use this functionality

Reply:
Will update later today and you can pull changes

Reply:
> Will update later today and you can pull changes

HI,

I sent an email.
thanks.
A.

Reply:
Will merge for now and delete later :)